exchange/bidder.go

```diff
@@ -87,6 +87,8 @@ type pbsOrtbSeatBid struct {
 	// if len(bids) > 0, this will become response.seatbid[i].ext.{bidder} on the final OpenRTB response.
 	// if len(bids) == 0, this will be ignored because the OpenRTB spec doesn't allow a SeatBid with 0 Bids.
 	ext json.RawMessage
+
+	warnings map[openrtb_ext.BidderName][]openrtb_ext.ExtBidderError
```

Comment:
Warnings are actually Errors. Should I use errortypes.Warning instead? If yes - I need to add Code in it:
```
type Warning struct {
	Message string
	Code int
}
```

Reply:
I agree it's a bit strange to see warnings as a collection of errors. This only a concern in code with the type name. I'm not sure its worth having two separate but identical data structures with different names, but I don't feel too strongly if you prefer to see `ExtBidderWarning`.

Reply:
It is reasonable to me that warnings are a subset of go "errors". If you need to define `Warning` in more than one package, then I would just add it to errortypes. If it appears in just one place, I don't have a strong feeling on it.

Reply:
Too be clear, I don't have an issue with `warnings map[openrtb_ext.BidderName][]openrtb_ext.ExtBidderError` appearing in multiple places, just the idea of defining a `Warning` struct multiple places.

Reply:
This new(or updated) Warning should also have json properties mapping.
```
type ExtBidderError struct {
	Code    int    `json:"code"`
	Message string `json:"message"`
}
```
Please advice. I'll do what you guys think is better.

Reply:
Another thing. Do we want to separate "models" from "views"? 
By model I mean `Warning` struct without json properties mapping, we use it "internally" only.
By "views" I mean another `Warning` struct we will Marshal to response. 
Now I use same `ExtBidderError` for everything.

Reply:
I lean towards just using `ExtBidderError` and using the `Code` value to determine the type. A "warning" can also be called a "non-fatal error", it is a problem, just not a problem severe enough to require aborting the processing. A new struct that is identical to an old struct should really just be used when type safety is an important concern. There is a large channel (communicating issues out to the requester) where we want to treat them interchangeably, to a degree, and just a few places where we want to separate them out. I am open to being convinced otherwise.

Reply:
I'm confused. 
`Code` in `ExtBidderError` is an error number like `UnknownWarningCode = 10999`, we need to set these values.
`Code` in `Warning` is Severity level. 
Do you mean I need to "consolidate" these structures?

Reply:
I reviewed again and I see the warnings are their own thing now in the code. I suppose it depends on future plans, if we want to reuse some of the error machinery for warnings in the future, then we should keep them consolidated. If this is the extent we want to have warnings, then we can consider making them their own struct.

Reply:
Per morning discussion I renamed ExtBidderError to more generic ExtBidderMessage. 
Should we leave it all as it is now?

Reply:
I am good with that.